ecs.tf

```diff
@@ -60,6 +60,7 @@ module "app_container_definition" {
   essential                = true
   readonly_root_filesystem = false
   environment              = var.environment_vars
+  secrets                  = "${var.secrets}"
```

Comment:
I don't think this string interpolation is required here.

Reply:
I believe so. We have tested this with the PR in carbon-zero-terraform pointing to this branch, and it seems to be working, although I've not yet removed the db-password in the circle-ci config if the feature flag repo

Reply:
Fixed